pkg/server/plugin/datastore/sql/mysql.go

```diff
@@ -18,23 +19,56 @@ import (
 	_ "github.com/spiffe/spire/pkg/server/plugin/datastore"
 )
 
-type mysql struct{}
+type mysqlDB struct{}
 
 const (
 	tlsConfigName = "spireCustomTLS"
 )
 
-func (my mysql) connect(cfg *configuration, isReadOnly bool) (*gorm.DB, error) {
+func (my mysqlDB) connect(cfg *configuration, isReadOnly bool) (*gorm.DB, string, bool, error) {
```

Comment:
Either a comment or named return values would be useful here.

Reply:
Agreed, it is unwieldy.